app/graphql/resolvers/farmer_resolver.py

```
@@ -0,0 +1,145 @@
+from __future__ import annotations
+from typing import Optional, List
+from uuid import UUID
+
+import strawberry
+from fastapi import HTTPException
+from sqlalchemy import select
+from strawberry.exceptions import StrawberryGraphQLError
+
+from app.core.database import get_db
+from app.graphql.types.farmer_type import Farmer, FarmerInput, FarmerUpdateInput
+from app.graphql.types.farmer_type import Location as GqlLocation
```

Ruff (0.12.2)

`app.graphql.types.farmer_type.Location` imported but unused

Remove unused import: `app.graphql.types.farmer_type.Location`

(F401)